Jamulus.pro

```diff
@@ -112,15 +104,26 @@ win32 {
             message("Warning: jack.h was not found in the expected location ($${programfilesdir}). Ensure that the right JACK2 variant is installed (32bit vs. 64bit).")
         }
 
-        HEADERS -= windows/sound.h
-        SOURCES -= windows/sound.cpp
         HEADERS += linux/sound.h
         SOURCES += linux/sound.cpp
         DEFINES += WITH_JACK
         DEFINES += JACK_REPLACES_ASIO
         DEFINES += _STDINT_H # supposed to solve compilation error in systemdeps.h
         INCLUDEPATH += "$${programfilesdir}/JACK2/include"
         LIBS += "$${programfilesdir}/JACK2/lib/$${libjackname}"
+    } else {
+        message(Using ASIO.)
```

Comment:
Might be worth adding a `message` about checking the ASIOSDK licence.

> Please review the ASIOSDK licence.

Reply:
Good idea, done.